internal/rest/avi_obj_vs.go

```diff
@@ -641,7 +641,27 @@ func (rest *RestOperations) AviVsVipBuild(vsvip_meta *nodes.AviVSVIPNode, cache_
 		vip := avimodels.Vip{AutoAllocateIP: &auto_alloc}
 		networkRef := lib.GetNetworkName()
 		if lib.IsPublicCloud() && lib.GetNetworkName() != "" {
-			vip.SubnetUUID = &networkRef
+
+			if lib.GetCloudType() == lib.CLOUD_GCP {
+				// add the IPAMNetworkSubnet
+				intCidr, err := strconv.ParseInt(lib.GetSubnetPrefix(), 10, 32)
+				if err != nil {
+					utils.AviLog.Warnf("The value of CIDR couldn't be converted to int32. Defaulting to /24")
+					intCidr = 24
```

Comment:
why are we defaulting to /24? Is this something documented in GCP? (if so can we cite it as code comment). Else it leads to some kind of an unknown behaviour.

Reply:
This seems to be the case in normal AKO workflow as well, where we default to /24 in case a subnetIP is provided but subnetPrefix is not provided or is invalid

Reply:
Is this documented anywhere? Maybe we should if not.

Reply:
I have taken note of this. Will add it in the helm docs repo.